.spin/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
This file is perhaps not needed. Is there actually nontrivial code that is copied straight from `spin`? IIRC that was the problem in NumPy, because `spin`'s interface was too inflexible and rather than extending commands we had to copy over code wholesale and then modify it.

Reply:
Deleted and I am resolving this thread.